native/src/main/java/io/ballerina/stdlib/protobuf/deserializers/DeserializeHandler.java

```diff
@@ -0,0 +1,286 @@
+                case DescriptorProtos.FieldDescriptorProto.Type.TYPE_FLOAT_VALUE: {
+                    new FloatDeserializer(input, fieldDescriptor, bMessage, targetType, isPacked).deserialize();
+                    break;
+                }
+                case DescriptorProtos.FieldDescriptorProto.Type.TYPE_INT64_VALUE: {
+                    new Int64Deserializer(input, fieldDescriptor, bMessage, targetType, isPacked).deserialize();
+                    break;
+                }
+                case DescriptorProtos.FieldDescriptorProto.Type.TYPE_UINT64_VALUE: {
+                    new UInt64Deserializer(input, fieldDescriptor, bMessage, targetType, isPacked).deserialize();
+                    break;
+                }
+                case DescriptorProtos.FieldDescriptorProto.Type.TYPE_INT32_VALUE: {
+                    new Int32Deserializer(input, fieldDescriptor, bMessage, targetType, isPacked).deserialize();
+                    break;
+                }
+                case DescriptorProtos.FieldDescriptorProto.Type.TYPE_UINT32_VALUE: {
+                    new UInt32Deserializer(input, fieldDescriptor, bMessage, targetType, isPacked).deserialize();
+                    break;
+                }
+                case DescriptorProtos.FieldDescriptorProto.Type.TYPE_FIXED64_VALUE: {
+                    new Fixed64Deserializer(input, fieldDescriptor, bMessage, targetType, isPacked).deserialize();
+                    break;
+                }
+                case DescriptorProtos.FieldDescriptorProto.Type.TYPE_FIXED32_VALUE: {
+                    new Fixed32Deserializer(input, fieldDescriptor, bMessage, targetType, isPacked).deserialize();
+                    break;
+                }
+                case DescriptorProtos.FieldDescriptorProto.Type.TYPE_BOOL_VALUE: {
+                    new BooleanDeserializer(input, fieldDescriptor, bMessage, targetType).deserialize();
+                    break;
+                }
+                case DescriptorProtos.FieldDescriptorProto.Type.TYPE_STRING_VALUE: {
+                    new StringDeserializer(input, fieldDescriptor, bMessage, targetType).deserialize();
+                    break;
+                }
+                case DescriptorProtos.FieldDescriptorProto.Type.TYPE_ENUM_VALUE: {
+                    new EnumDeserializer(input, fieldDescriptor, bMessage, targetType).deserialize();
+                    break;
+                }
+                case DescriptorProtos.FieldDescriptorProto.Type.TYPE_BYTES_VALUE: {
+                    new BytesDeserializer(input, fieldDescriptor, bMessage, targetType).deserialize();
+                    break;
+                }
+                case DescriptorProtos.FieldDescriptorProto.Type.TYPE_MESSAGE_VALUE: {
+                    new MessageDeserializer(input, fieldDescriptor, messageType, bMessage, targetType)
+                            .deserialize();
+                    break;
+                }
+                default: {
```

Comment:
we can remove this right?

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-protobuf/pull/176/commits/4c2cc7dfa823b33f8e641a6c70a9e475c7621235